include/boost/geometry/algorithms/detail/disjoint/multipoint_geometry.hpp

```diff
@@ -378,14 +400,15 @@ class multi_point_multi_geometry
         template <typename Point, typename BoxPair>
         inline bool apply(Point const& point, BoxPair const& box_pair)
         {
-            typedef typename PtSegStrategy::disjoint_point_box_strategy_type d_pb_strategy_type;
-
             typedef typename boost::range_value<MultiGeometry>::type single_type;
 
             // The default strategy is enough for Point/Box
             if (! m_intersection_found
-                && ! detail::disjoint::disjoint_point_box(point, box_pair.first, d_pb_strategy_type())
-                && ! dispatch::disjoint<Point, single_type>::apply(point, range::at(m_multi_geometry, box_pair.second), m_strategy))
+                && ! detail::disjoint::disjoint_point_box(point, box_pair.first, m_strategy)
+                && ! dispatch::disjoint
+                        <
```

Comment:
4 spaces of indentation here, right?

Reply:
It'd look bad:
```
&& ! dispatch::disjoint
    <
        Point, single_type
    >::apply(point, range::at(m_multi_geometry, box_pair.second), m_strategy))
```

In such cases I put as many tabs/spaces as it is needed to start the arguments list after the name of the function/struct starts.

Reply:
OK